tests/functionaltests/Tests/UIKitTests/UIButtonTests.mm

```diff
@@ -1411,21 +1413,23 @@ - (void)dealloc {
         __block auto uxEvent = UXEvent::CreateAuto();
         __block auto xamlSubscriber = std::make_shared<XamlEventSubscription>();
         __block NSString* expectedBackgroundImage = @"150x150.png";
+        EventRegistrationToken ert{};
+        EventRegistrationToken* const ertAddr = &ert;
 
         WXFrameworkElement* xamlElement = [buttonToTest xamlElement];
         ASSERT_OBJCNE(xamlElement, nil);
 
         // Wait for the layerContent to be part of the visual tree
         dispatch_sync(dispatch_get_main_queue(), ^{
-            EventRegistrationToken ert = [xamlElement addLayoutUpdatedEvent:^(RTObject* sender, RTObject* args) {
+            *ertAddr = [xamlElement addLayoutUpdatedEvent:^(RTObject* sender, RTObject* args) {
```

Comment:
Does __block tunnel through multiple levels of blocks? I tried that first and the test was still crashing, so I assumed that it didn't, but I don't know for sure how it's supposed to work.